src/main/java/me/coley/recaf/config/ConfDisplay.java

```diff
@@ -57,6 +57,11 @@
 	 */
 	@Conf("display.maxrecent")
 	public long maxRecentFiles = 6;
+	/**
+	 * Use system menubar if it is possible.
```

Comment:
As mentioned below, may want to mention that this is a mac-only feature in the comment here.